templates/web/base/report/_item.html

```diff
@@ -43,9 +43,13 @@
   [% PROCESS 'report/_item_extra_class.html' %]
 [%~ CATCH file ~%]
 [%~ END ~%]
-
 <li class="item-list__item item-list--reports__item [% item_extra_class %]"
     data-report-id="[% problem.id | html %]" data-lastupdate="[% problem.lastupdate %]" id="report-[% problem.id | html %]">
+[% IF relevant_staff AND (c.user.has_permission_to('assign_report_to_user', problem.bodies_str_ids)) %]
+    [% UNLESS ((c.req.uri.path.match('\/my\/')) OR  (c.req.uri.path.match('\/around'))) %]
```

Comment:
There is a `page` stash variable (used elsewhere in this template) you can check being equal to `around` or `my`.